DataTables.Blazor/wwwroot/datatables.blazor.js

```diff
@@ -136,6 +139,19 @@ function addRowClickListener(tableElement, dotNetCallback) {
     });
 }
 
+function addRowDoubleClickListener(tableElement, dotNetCallback) {
+    $(tableElement).on("dblclick", "tbody tr", function (...args) {
+        var row = $(tableElement).DataTable().row(this);
+        var json = {
+            "index": row.index(),
+            "id": String(row.id()),
+            "data": row.data()
+        };
+        console.log(json);
```

Comment:
Was this just debugging? Or is there a reason you wanted to leave it in?

Reply:
Yes, console.log was just for debugging purpose, I made a commit to fix it.